Gemfile

```diff
@@ -41,6 +41,11 @@ group :development, :test do
   gem 'pry'
   gem 'capybara'
```

Comment:
I think capybara, rspec-rails, shoulda-matchers and simplecov should be in the test block and not needed in development?

Reply:
That sounds good.

Reply:
Same thing with Launchy.